lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -525,6 +533,28 @@ def generate_clone_spec(src_config)
       clone_spec.config.memoryMB = Integer(get_config(:customization_memory)) * 1024
     end
 
+    if get_config(:customization_macs)
+      unless get_config(:customization_ips)
+        abort('Must specify IP numbers with --cips when specifying MAC addresses with --cmacs, can use "dhcp" as placeholder')
```

Comment:
Thank you *so* much for pointing the admin to the next step if they try to use this incorrectly.